tests/test_commit_message.py

```diff
@@ -36,6 +36,58 @@ def test_commit_message_reviewers_replaced():
     assert commit_message == (FIRST_LINE, COMMIT_MESSAGE)
 
 
+@pytest.mark.parametrize(
+    "reviewers",
+    [
+        ["a"],
+        ["aa"],
+        [".a"],
+        ["..a"],
+        ["...a"],
+        ["a...a"],
+        ["a.b"],
+        ["a.b.c"],
+        ["-.-.-"],
+        ["a", "aa", ".a", "..a", "...a", "a...a", "a.b", "a.b.c", "-.-.-"],
+    ],
+)
+def test_various_reviewer_name_combinations(reviewers):
+    """Tests single and multiple characters separated by various combinations
+    of periods for single and multiple reviewers.
```

Comment:
nit: we follow [PEP 257](https://www.python.org/dev/peps/pep-0257) for our docstrings. Specifically your new tests violate...
> The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.

...from [PEP 257 #multi-line-docstrings](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)